### PR TITLE
react-mentions-click types

### PR DESCRIPTION
**Pull request by AkashLakade**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:

- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:

- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

If removing a declaration:

- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Review by AkashLakade05 — APPROVED** [review]

**Comment by RyanCavanaugh** [comment]
Aside: having your alt account approve a PR you wrote is going to make everyone *extremely* suspicious that you're up to no good; I really recommend not doing that here or anywhere else for that matter.

File: types/react-mentions-click/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "@typescript-eslint/no-empty-interface": "off"
```
Comment (RyanCavanaugh) [inline types/react-mentions-click/.eslintrc.json:3]: Don't globally disable this rule. It's there for a reason. If you need to violate it, document why in-situ

File: types/react-mentions-click/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "private": true,
+    "name": "@types/react-mentions-click",
+    "version": "1.0.9999",
+    "projects": [
+        "https://github.com/signavio/react-mentions"
```
Comment (RyanCavanaugh) [inline types/react-mentions-click/package.json:6]: This should be `react-mention-clicks`'s repo

Comment (RyanCavanaugh) [reply types/react-mentions-click/package.json:6]: What is the originating repo? I can find search hits for https://github.com/AkashLakade05/react-mentions-click but that's deleted.

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

**Comment by RyanCavanaugh** [comment]
This doesn't seem to be a public npm package and does seem to be authored by you. Closing since this is not an in-scope PR and seems to possibly be fishy behavior.

**Pull request closed without merging**